template/rwa-onboarding/Goerli-DssSpell.sol

```diff
@@ -35,7 +35,14 @@ contract DssSpellAction is DssAction, DssSpellCollateralAction {
     }
 
     function actions() public override {
+        // ---------------------------------------------------------------------
+        // Includes changes from the DssSpellCollateralAction
         onboardNewCollaterals();
+
+        // lock RWA999 Token in the URN
+        GemLike(RWA999).approve(RWA999_A_URN, 1 * WAD);
+        RwaUrnLike(RWA999_A_URN).lock(1 * WAD);
```

Comment:
I don't think this part should be in the template, because this happens on a per-deal basis.

Reply:
done